docs/api/native-image.md

```diff
@@ -133,6 +133,13 @@ Returns a [Buffer][buffer] that contains the image's `JPEG` encoded data.
 
 Returns the data URL of the image.
 
+### `image.asNativeRepresentation(type)`
+
+Returns a pointer to an underlying native type (encoded as a [Buffer][buffer]) which can be used with native APIs. Note that in many cases, this pointer is a weak pointer to the underlying native image not a copy, so you _must_ ensure that the associated `nativeImage` instance is kept around.
+
+* `type` String (**required**) - one of:
```

Comment:
So on Windows you have at least `HICON` and `HBITMAP`, and I imagine there might be more depending on what people want